classes/auto-package-utils.oeclass

```diff
@@ -63,24 +63,36 @@ def auto_package_utils (d):
         else:
             utilname = util.replace("_", "-").replace(".", "-").lower()
         pkg = "%s-%s"%(basename, utilname)
-        utilname = "util/" + utilname
+        symlinks = (d.get("AUTO_PACKAGE_UTILS_SYMLINKS_" + utilname) or "").split()
         docpkg = pkg + "-doc"
         packages += [ pkg ]
         doc_packages += [ docpkg ]
-        provides += [ utilname ]
-
-        util = prefix + util
-        files = ("${base_sbindir}/%s "%(util) +
-                 "${base_bindir}/%s "%(util) +
-                 "${sbindir}/%s "%(util) +
-                 "${bindir}/%s "%(util) +
-                 "${libexecdir}/%s "%(util))
-        if exeext:
-            files += (" ${base_sbindir}/%s%s "%(util, exeext) +
-                      "${base_bindir}/%s%s "%(util, exeext) +
-                      "${sbindir}/%s%s "%(util, exeext) +
-                      "${bindir}/%s%s "%(util, exeext) +
-                      "${libexecdir}/%s%s "%(util, exeext))
+
+        pkg_provides = [ "util/" + utilname ]
+        basenames = [ prefix + util ]
+
+        for s in symlinks:
+            if ":" in s:
+                sname, s = s.split(":", 1)
+                sname = sname.replace("_", "-").replace(".", "-").lower()
+            else:
+                sname = s.replace("_", "-").replace(".", "-").lower()
+            pkg_provides.append("util/" + sname)
+            basenames.append(prefix + s)
+
+        files = ""
+        for b in basenames:
+            files += ("${base_sbindir}/%s "%(b) +
+                      "${base_bindir}/%s "%(b) +
+                      "${sbindir}/%s "%(b) +
+                      "${bindir}/%s "%(b) +
+                      "${libexecdir}/%s "%(b))
+            if exeext:
+                files += (" ${base_sbindir}/%s%s "%(b, exeext) +
+                          "${base_bindir}/%s%s "%(b, exeext) +
+                          "${sbindir}/%s%s "%(b, exeext) +
+                          "${bindir}/%s%s "%(b, exeext) +
+                          "${libexecdir}/%s%s "%(b, exeext))
```

Comment:
no idea, just copied what was already there for the util itself. Does windows even have the symlink concept?